### DISCUSSION TITLE
2FA via One Time Password via mobile

### DISCUSSION THREAD

**Discussion by ThoWi-00** [netbox-community/netbox#11852] [Help Wanted!] [answered]
Dear Community,

I installed Netbox last week as discribed via Docker. It works fine and all looks good.
Now I want to share the data with my friends/colleagues over the Internet but I'm not sure wether I can secure the access via 2FA OTP.
I found a plug-in on github (https://github.com/k1nky/netbox-otp-plugin), but I think this works only with a manual installation of Netbox.

Is there any way to implement this functionallty or is this feature maybe already implemented?

Thanks for any hint.

Best regards,

ThoWi-00

**Comment by candlerb** [comment, marked as answer]
To use the netbox-docker image with plugins, you have to build your own image with the plugins added.

See: https://github.com/netbox-community/netbox-docker/wiki/Using-Netbox-Plugins

**Reply by ThoWi-00** [reply]
Hi Candlerb,

thanks for the fast answer. 
I'm not so familiar with docker installations, but I'll try :-)

Thanks.
